eventlet/wsgi.py

```diff
@@ -446,23 +458,23 @@ def handle_one_request(self):
                 self.close_connection = 1
                 return
 
-        self.environ = self.get_environ()
+        headers_sent = []
```

Comment:
I'm not entirely sure whether I like moving this up and out to `handle_one_request`, but I didn't see a good way to plumb the flag down into `Input` otherwise. If anyone else has a better idea, I'm all ears...